### ISSUE TITLE
BMSHJ2018 Parameters

### ISSUE THREAD

**Issue by alexhepburn** [InterDigitalInc/CompressAI#2]
I was wondering if the training parameters for both `'bmshj2018_factorized'` & `'bmshj2018_hyperprior'` were anywhere in the documentation? E.g. the dataset, optimisation, learning rates, training epochs ect.

I am assuming that the `quality` parameter loads different pre-trained networks depending on the lambda parameter they were trained with in the rate-distortion loss. It would be good to what lambda corresponds to the `quality` parameter.
Thanks!

**Comment by alexhepburn** [comment]
Sorry, I completely missed this in the documentation, found here: https://interdigitalinc.github.io/CompressAI/zoo.html#training
